### PR TITLE
Fix login issue in console and myaccount

### PR DESCRIPTION
**Pull request by yathindrakodithuwakku**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/webpack.config.js [hunk 1]
```diff
@@ -518,6 +512,58 @@ module.exports = (env) => {
...
+                        : "",
+                    importTenantPrefix: !isDeployedOnExternalServer
+                        ? "<%@ page import=\"static org.wso2.carbon.utils.multitenancy." +
+                        "MultitenantConstants.TENANT_AWARE_URL_PREFIX\"%>"
+                        : "",
+                    importUtil: !isDeployedOnExternalServer
+                        ? "<%@ page import=\"" +
+                        "static org.wso2.carbon.identity.core.util.IdentityUtil.getServerURL\" %>"
+                        : "",
+                    clientID: "CONSOLE",
```
Comment (brionmario) [inline apps/console/webpack.config.js:537]:
```suggestion
                    clientID: deploymentConfig.clientId,
```

File: apps/console/src/index.jsp [hunk 2]
```diff
@@ -36,13 +30,11 @@
...
 
-            var userTenant = userAccessedPath.split("/t/")[1] ?  userAccessedPath.split("/t/")[1].split("/")[0] : null;
-            userTenant = userTenant ?  userTenant.split("?")[0] : null;
-
-            var serverOrigin = "<%=serverUrl%>";
-            var authorizationCode = "<%=authorizationCode%>" != "null" ? "<%=authorizationCode%>" : null;
-            var authSessionState = "<%=authSessionState%>" != "null" ? "<%=authSessionState%>" : null;
-            var authIdPs = "<%=authIdPs%>" != "null" ? "<%=authIdPs%>" : null;
+            var serverOrigin = "<%= htmlWebpackPlugin.options.serverUrl %>";
+            var authorizationCode = "<%= htmlWebpackPlugin.options.authorizationCode %>" != "null" ?
```
Comment (brionmario) [inline apps/console/src/index.jsp:34]: Shall we format for readability.
```suggestion
            condition
                ? true path
                : false path
```

File: apps/console/src/index.jsp [hunk 3]
```diff
@@ -55,45 +47,32 @@
...
                 enablePKCE: true,
                 overrideWellEndpointConfig: true
             }
 
             var isSilentSignInDisabled = userAccessedPath.includes("disable_silent_sign_in");
-            var isTenantSwitchPath = userAccessedPath.includes("switch_tenant");
+            var isSignOutSuccess = userAccessedPath.includes("sign_out_success");
+
+            if(isSignOutSuccess) {
+                window.location.href = applicationDomain+'/'+"<%= htmlWebpackPlugin.options.basename %>"
```
Comment (inthirakumaaran) [inline apps/console/src/index.jsp:65]:
```suggestion
                window.location.href = applicationDomain + '/' + "<%= htmlWebpackPlugin.options.basename %>"
```

File: apps/console/src/index.jsp [hunk 4]
```diff
@@ -55,45 +47,32 @@
...
             
             if(isSilentSignInDisabled) {
-
-                if(isTenantSwitchPath) {
-                    auth.initialize(authConfig);
-                    auth.signIn();
-                } else {
-                    window.location.href = applicationDomain + '/authenticate?disable_silent_sign_in=true&invite_user=true';
-                }
+                window.location.href = applicationDomain + '/'+"<%= htmlWebpackPlugin.options.basename %>"+'/authenticate?disable_silent_sign_in=true&invite_user=true';
```
Comment (inthirakumaaran) [inline apps/console/src/index.jsp:69]:
```suggestion
                window.location.href = applicationDomain + '/' + "<%= htmlWebpackPlugin.options.basename %>" + '/authenticate?disable_silent_sign_in=true&invite_user=true';
```

File: apps/console/src/index.jsp [hunk 5]
```diff
@@ -55,45 +47,32 @@
...
             } else {
 
                 if(authorizationCode) {
                     sessionStorage.setItem("userAccessedPath", userAccessedPath.split(window.origin)[1]);
 
-                    window.location.href = applicationDomain+'/authenticate?code='+authorizationCode+'&state=sign-in-silently'+'&AuthenticatedIdPs='+authIdPs+
+                    window.location.href = applicationDomain+'/'+"<%= htmlWebpackPlugin.options.basename %>"+'/authenticate?code='+authorizationCode+
```
Comment (inthirakumaaran) [inline apps/console/src/index.jsp:77]:
```suggestion
                    window.location.href = applicationDomain + '/' + "<%= htmlWebpackPlugin.options.basename %>" + '/authenticate?code=' + authorizationCode +
```

Comment (inthirakumaaran) [reply apps/console/src/index.jsp:77]: Check other places for formatting issues

File: apps/console/src/index.jsp [hunk 6]
```diff
@@ -55,45 +49,32 @@
...
-                    window.location.href = applicationDomain + '/authenticate?disable_silent_sign_in=true&invite_user=true';
-                }
+                window.location.href = applicationDomain + '/' + "<%= htmlWebpackPlugin.options.basename %>" + '/authenticate?disable_silent_sign_in=true&invite_user=true';
             } else {
 
                 if(authorizationCode) {
                     sessionStorage.setItem("userAccessedPath", userAccessedPath.split(window.origin)[1]);
 
-                    window.location.href = applicationDomain+'/authenticate?code='+authorizationCode+'&state=sign-in-silently'+'&AuthenticatedIdPs='+authIdPs+
+                    window.location.href = applicationDomain + '/' + "<%= htmlWebpackPlugin.options.basename %>" + '/authenticate?code=' + authorizationCode+
```
Comment (inthirakumaaran) [inline apps/console/src/index.jsp:77]:
```suggestion
                    window.location.href = applicationDomain + '/' + "<%= htmlWebpackPlugin.options.basename %>" + '/authenticate?code=' + authorizationCode +
```

**Review by brionmario — APPROVED** [review]

**Pull request merged**